_rules/remap-shortcut-aa8b52.md

```diff
@@ -0,0 +1,225 @@
+---
+id: aa8b52
+name: Keyboard shortcut can be remapped
+rule_type: atomic
+
+description: |
+  This rule checks that if keyboard shortcuts using only letter, number, sign or punctuation keys are implemented there is a mechanism to remap the shortcut to use one or more non-printable keyboard characters
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.1.4: # Character Key Shortcuts (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+
+authors:
+  - João Vicente

```

Comment:
You may have to add yourself to the contributors list in `package.json`. Please do this in a different PR.

`npm test` will fail otherwise reporting a missing author.